app/memory/context_manager.py

```
@@ -0,0 +1,422 @@
+"""Context management system for maintaining conversation state and learning from interactions."""
+
+import json
+import os
```

Ruff (0.11.9)

`os` imported but unused

Remove unused import: `os`

(F401)